src/main/java/net/mcreator/element/types/Tag.java

```diff
@@ -35,11 +35,11 @@
 
 	@Nonnull public String type;
 
-	public List<MItemBlock> items;
-	public List<MItemBlock> blocks;
+	@ModElementReference public List<MItemBlock> items;

```

Comment:
Why can't those be automatically looked up as MappableElement?

Reply:
Type parameters are not scanned. To get rid of these additional marks, I'd need to restore the previous behavior when all fields were checked unconditionally.

Reply:
Ok, probably type erasure, nevermind